snapd-glib/snapd-autorefresh-change-data.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2024 Canonical Ltd.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2 or version 3 of the License.
+ * See http://www.gnu.org/copyleft/lgpl.html the full text of the license.
+ */
+#pragma once
+
+#include <glib-object.h>
+#include "snapd-change-data.h"
+
+G_BEGIN_DECLS
+
+#define SNAPD_TYPE_AUTOREFRESH_CHANGE_DATA  (snapd_autorefresh_change_data_get_type ())
+G_DECLARE_FINAL_TYPE (SnapdAutorefreshChangeData, snapd_autorefresh_change_data, SNAPD, AUTOREFRESH_CHANGE_DATA, SnapdChangeData)
+
+GStrv     snapd_autorefresh_change_data_get_snap_names         (SnapdAutorefreshChangeData *change_data);
```

Comment:
Existing code used gchar ** instead of GStrv here - GStrv doesn't seem to be used for return values, though I'm not sure why that is (just historical?)

Reply:
Actually checking shows we are using GStrv, ignore this.